nestkernel/connection_manager.cpp

```diff
@@ -195,38 +201,40 @@ nest::ConnectionManager::set_synapse_status( index gid,
 void
 nest::ConnectionManager::delete_connections_()
 {
-  for ( tVSConnector::iterator it = connections_.begin();
-        it != connections_.end();
-        ++it )
+#ifdef _OPENMP
+#pragma omp parallel
   {
-    for ( tSConnector::nonempty_iterator iit = it->nonempty_begin();
-          iit != it->nonempty_end();
-          ++iit )
+#pragma omp for schedule( static, 1 )
```

Comment:
well, we are using the thread id in many places to directly access `connections_[ tid ]` in an `omp parallel` block, as you point out, so this assignment should also hold here. i agree with you that we should think about our threading model in nest, but this might be beyond the scope of this PR.